### ISSUE TITLE
New direction + maintainer

### ISSUE THREAD

**Issue by lukef** [lukepighetti/fluro#184]
Good news. Fluro is moving to a new home. 

Say hello to @lukepighetti! Luke is one of the most active, enthusiastic and talented Flutter developers there is. I'm glad it's going to a friend.

I won't get too deep into it but some of the policies adopted by the Flutter team have made it pretty much untenable for Yakka to personally maintain or develop this library. Constant breaking changes and, what we consider to be, generally less than ideal community consultation / collaboration have taken all the "fun" out of providing Fluro to you all.

I know many of you have already moved on or become frustrated with lack of communication and progress. I've, personally, obsessed over what to do with the project over many months. Many times I thought of shutting it down. However, I think there is still **plenty** life left in Fluro! There are still lots of cool ideas and everyone is excited for it to move way beyond what we envision.

Thanks for all your support! 

Luke

**Comment by lukepighetti** [comment]
Hi everyone! I'm taking over the `fluro` package from Luke. Two Lukes? I know, it's a little confusing, but the good news is you don't have to learn any new names!

So what's happening with `fluro`? Good question!

1. Fix the `Router` class name collision and do a patch release on pub.
2. Triage all the issues and knock out a minor release with a handful of fixes.
3. Start working on a major release to transition `fluro` to be a more opinionated all-around solution making use of the new Pages / Navigator 2.0 API.

**Comment by lukepighetti** [comment]
Hi everyone! I'm taking over the `fluro` package from @lukef. Two Lukes? I know, it's a little confusing, but the good news is you don't have to learn any new names!

So what's happening with `fluro`? Good question!

1. Fix the `Router` class name collision and do a patch release on pub.
2. Triage all the issues and knock out a minor release with a handful of low-hanging-fruit fixes.
3. Start working on a major release to transition `fluro` to the new Pages / Navigator 2.0 API.

**Comment by lukepighetti** [comment]
First two items have been resolved and Fluro `1.7.0` is now on pub! https://pub.dev/packages/fluro

**Comment by chiragmittal19** [comment]
Great to see Fluro getting back into business. Have already used this powerful library in multiple projects. Looking forward to more features. Wish you the best!

**Comment by li8607** [comment]
赞

**Comment by teneon** [comment]
Hi Luke Pighetti,

i am so glad someone took over Fluro. It could become one of the most important packages for Flutter! Wish you all the best!

**Comment by esarbanis** [comment]
Hey @lukepighetti , glad to see someone taking over Fluro 💪 , perhaps one of my favorite libraries! As I am quite invested in it for my projects, if there is anything I can help out with, please let me know.

**Comment by lukepighetti** [comment]
Quick update! I've been wanting to get this migrated over to Nav 2.0 / Pages API for a while now, but I'm still waiting on core Flutter to get a little more developed in this area before making the switch. I just learned today that built-in deeplinking has been added to Nav 2.0 and is currently in Flutter `master` channel. Once this hits beta I will have another look at the state of the Nav 2.0 API and see if it's mature enough for us to do a rethink/rewrite of Fluro to leverage the fully declarative Nav 2.0 API.
